test/src/test/java/hudson/model/AbstractBuildTest.java

```diff
@@ -320,10 +320,6 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
 
                 return true;
             }
-
-            private Object writeReplace() {
```

Comment:
https://github.com/openrewrite/rewrite-static-analysis/commit/976d366036dc785107f199d8a3467a2675959469


1to1 the same phrase:

`private Object writeReplace`

Reply:
https://github.com/openrewrite/rewrite-static-analysis/commit/7a9acb8ee5a79857494b66d4e290da9858a9f251